client/my-sites/plans-features-main/index.tsx

```diff
@@ -239,6 +239,7 @@ const PlansFeaturesMain = ( {
 	const { setShowDomainUpsellDialog } = useDispatch( WpcomPlansUI.store );
 	const domainFromHomeUpsellFlow = useSelector( getDomainFromHomeUpsellInQuery );
 	const showUpgradeableStorage = config.isEnabled( 'plans/upgradeable-storage' );
+	const shouldUseSmallestUnitCurrency = flowName === 'onboarding';
```

Comment:
**Note:**

We'd like to scope the smallestUnit currency changes to the `/start` flow for minimal impact. If other folks have suggestions about better ways to limit the surface area of the changes, feel free to chime in 🙂